.github/workflows/build.yaml

```diff
@@ -1,33 +1,44 @@
 name: "build"
 on: [push, pull_request]
+
 jobs:
   build:
     name: build
     runs-on: ubuntu-latest
     steps:
       - name: Setup BATS
```

Comment:
How about using actions/checkout instead of using `git clone`?
```
      - name: Checkout BATS
        uses: actions/checkout@v4
        with:
          repository: bats-core/bats-core
          ref: v1.11.0
```